observability/prometheus/rules/rabbitmq/README.md

```diff
@@ -0,0 +1,13 @@
+# RabbitMQ Prometheus Rules
+
+## Adding new Prometheus Rules
+
+To allow filtering in the [RabbitMQ-Alerts Grafana dashboard](../../../grafana/dashboards/rabbitmq-alerts.yml) and allowing [configuration of Alertmanager](../../alertmanager) all RabbitMQ alerts must output at least the labels:
+* `rulesgroup: rabbitmq`
+* `namespace`
+* `rabbitmq_cluster`
+* `severity`
```

Comment:
None of these rules have all of these; am I missing something?

Reply:
All rules output all these labels. In some rules, they are not explicitly added via the `labels` field but implicitly output by the PromQL expression. I added a sentence to the README which makes it clearer.